internal/blobcache/blobcache.go

```diff
@@ -11,29 +11,29 @@ import (
 
 // Crude estimate of the overhead per blob: a SHA-256, a linked list node
 // and some pointers. See comment in blobCache.add.
-const cacheOverhead = len(restic.ID{}) + 64
+const CacheOverhead = len(restic.ID{}) + 64
 
 // A blobCache is a fixed-size cache of blob contents.
 // It is safe for concurrent access.
-type blobCache struct {
+type BlobCache struct {
 	mu sync.Mutex
 	c  *simplelru.LRU
 
 	free, size int // Current and max capacity, in bytes.
 }
 
 // Construct a blob cache that stores at most size bytes worth of blobs.
-func newBlobCache(size int) *blobCache {
-	c := &blobCache{
+func NewBlobCache(size int) *BlobCache {
```

Comment:
blobcache.NewBlobCache is repetitive. Please rename it New.

Reply:
ack, updated in current version